drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -45,7 +45,8 @@ function DragNDropTemplates(url_name) {
             'draggable': !item.drag_disabled,
             'aria-grabbed': item.grabbed,
             'data-value': item.value,
-            'data-drag-disabled': item.drag_disabled
+            'data-drag-disabled': item.drag_disabled,
+            'aria-describedby': gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.'),
```

Comment:
@dovinmu yeah, that makes sense to me.  aria-describedby can take a space separated list of IDrefs, so you might be able to just add/remove IDrefs as necessary. They are read to the user in order, so I think the keyboard usage should be at the end, that way they get the most relevant information first (after a few uses they will be aware of how to use with a keyboard.)  Make sense?